tests/e2e/pageObjects/RegisterPage.js

```diff
@@ -1,36 +1,39 @@
 require("dotenv").config();
+
 const config=require("../../cucumber.conf.js")
+
 class RegisterPage {
+
   constructor() {
     this.baseUrl = config.tandoorURL;
     this.setupURL = this.baseUrl + "setup/";
-    this.username = `//input[@name="name"]`;
-    this.password = `#id_password`;
-    this.cPassword = `//input[@name="password_confirm"]`;
-    this.registerBtn = `//button[@class="btn btn-success"]`;
-    this.errorMsg = `//div[@id="div_id_password"]/div`;
-    this.succesMsg = `//div[@class="alert alert-success alert-dismissible fade show"]`;
+    this.usernameSelector = `//input[@name="name"]`;
+    this.passwordSelector = `#id_password`;
+    this.cPasswordSelector = `//input[@name="password_confirm"]`;
+    this.registerBtnSelector = `//button[@class="btn btn-success"]`;
+    this.errorMsgSelector = `//div[@id="div_id_password"]/div`;
```

Comment:
if a more specific selector can be found then use it. This selector can cause problem in the future (might resolve to multiple elements)
```suggestion
    this.errorMsgSelector = `//div[@id="div_id_password"]/div`;
```

Reply:
> if a more specific selector can be found then use it. This selector can cause problem in the future (might resolve to multiple elements)

multiple errors are excepted